packages/yourloops/components/menus/team-menu.tsx

```diff
@@ -107,11 +111,29 @@ function TeamMenu(): JSX.Element {
     setTeamCreationDialogData(null);
   };
 
-  const createCareTeam = () => {
-    setTeamCreationDialogData({ team: null, onSaveTeam });
+  const onTeamMenuTeamsAction = () => {
+    if (isUserHcp) {
+      setTeamCreationDialogData({ team: null, onSaveTeam });
```

Comment:
Can we move "onSaveTeam" into the dialog component ? These props are weird. I think it must be the dialog itself that handles saving, update etc...